ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -738,6 +738,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnected-sdn-workers-rhcos-rhel8
+- as: aws-ipi-hypershift-install-test-p1
```

Comment:
@LiangquanLi930 each of your case will create one cluster. although it is serial, it totally take at least two cluster at same time. Frankly it is cost of resource.
please indicate it to qe-producitivity team to confirm if it is OK for resource (I ask it because maybe qe-productivity team does not know how your case cost resource).

cc @liangxia @jhou1

Reply:
@LiangquanLi930 it is still P1 and do not see the confirmation, so I can not add lgtm to it because it is out of my control.

Reply:
@liangxia Could help to review it? From hypershift QE team, we want this case can be run 2--3 times a week, but currently, there is no choice, I know you have one PR to merge later. Currently, we can only choose P1. later on, we can update it according to your PR.

Reply:
Ack